r/vignettes/fs.Rmd

```diff
@@ -13,7 +13,11 @@ parts of the project to be able to read and write data with different storage
 backends. In the `arrow` R package, support has been enabled for AWS S3.
 This vignette provides an overview of working with S3 data using Arrow.
 
-> In Windows and macOS binary packages, S3 support is included. On Linux when installing from source, S3 support is not enabled by default, and it has additional system requirements. See `vignette("install", package = "arrow")` for details.
+> In Windows and macOS binary packages, S3 support is included. On Linux when 
```

Comment:
This came up in an issue the other day - S3 support is only available with certain versions of R.  It'd be good to check if the R>3.6 requirement is Windows-specific too.

```suggestion
> In Windows (for R > 3.6) and macOS binary packages, S3 support is included. On Linux when 
```